scripts/transpilation/copy-utils.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require("fs");
+const { execSync } = require('child_process');
+
+// eslint-disable-next-line no-console
+console.log('Copying bpk-mixins...');
+
+// if(!fs.existsSync('dist/bpk-mixins')) {
```

Comment:
do we need this?

Reply:
Ooop good catch!

Reply:
The copy command already handle this processing and creating it if its not there, otherwise it makes nested folder which is bad!

Reply:
Oh I thought the `cp` command would fail if dest folder doesn't exist 🤔